capacity4more/modules/c4m/restful/c4m_restful_quick_post/components/c4m-app/src/controllers/main.js

```diff
@@ -321,10 +321,6 @@ angular.module('c4mApp')
       // Check for required fields.
       var errors = Request.checkRequired(submitData, resource, resourceFields);
 
-      // Check the type of the submit.
-      // Make node unpublished if requested to create in full form.
-      submitData.status = type == 'full_form' ? 0 : 1;
```

Comment:
The important thing is that other users won't see unfinished posts (Started editing in full form and left without finishing), We have to find a way to do that.